DigitalLearningSolutions.Web/Views/SuperAdmin/Faqs/SuperAdminFaqs.cshtml

```diff
@@ -0,0 +1,44 @@
+@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
+@using Microsoft.Extensions.Configuration
+@model FaqsPageViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/superAdmin/faqs.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "FAQs";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter">
+    <nav class="side-nav-menu" aria-label="Side navigation bar">
+      <partial name="~/Views/SuperAdmin/Shared/_SideNavMenu.cshtml" model="Model" />
+    </nav>
```

Comment:
Subtle thing for side nav menus - these were originally taken from the NHS style guide pages, and like those pages, the side nav menu should be shifted to underneath the main content when the screen size is small enough. To do this, we just throw the partial underneath the main content as well, but with slightly different html attributes. This allows us to hide whichever one we don't need (taken care of with the classes)

See the TrackingSystem/Delegates/AllDelegates/Index.cshtml file for an example.